iocore/net/SSLUtils.cc

```diff
@@ -1550,17 +1697,17 @@ SSLMultiCertConfigLoader::_store_ssl_ctx(SSLCertLookup *lookup, const shared_SSL
     return false;
   }
 
-  if (!common_names.empty()) {
-    shared_SSL_CTX ctx(this->init_server_ssl_ctx(data, sslMultCertSettings.get(), common_names), SSL_CTX_free);
-
-    if (!ctx || !sslMultCertSettings || !this->_store_single_ssl_ctx(lookup, sslMultCertSettings, ctx, common_names)) {
+  std::vector<SSLLoadingContext> ctxs = this->init_server_ssl_ctx(data, sslMultCertSettings.get(), common_names);
+  for (const auto &loadingctx : ctxs) {
+    shared_SSL_CTX ctx(loadingctx.ctx, SSL_CTX_free);
+    if (!sslMultCertSettings || !this->_store_single_ssl_ctx(lookup, sslMultCertSettings, ctx, loadingctx.ctx_type, common_names)) {
       retval = false;
       std::string names;
-      for (auto name : data.cert_names_list) {
+      for (auto const &name : data.cert_names_list) {
         names.append(name);
         names.append(" ");
       }
-      Warning("(%s) Failed to insert SSL_CTX for certificate %s entries for names already made", this->_debug_tag(), names.c_str());
+      Warning("(%s) Failed to insert SSL_CTX for certificate %sentries for names already made", this->_debug_tag(), names.c_str());
```

Comment:
Lost a space between %s and entries

Reply:
I did it on purpose(message ends up having double space in log message), but not worth worrying about